RecoTracker/PixelLowPtUtilities/src/StripSubClusterShapeTrajectoryFilter.cc

```diff
@@ -265,74 +265,84 @@ bool StripSubClusterShapeFilterBase::testLastHit(const TrackingRecHit *hit,
       return true;
     }
 
+    bool allSame = std::all_of(std::begin(ampls), std::end(ampls), [&](int i) { return i == ampls[0]; });
```

Comment:
```suggestion
    bool allSame = std::all_of(std::begin(ampls), std::end(ampls), [&](uint8_t i) { return i == ampls[0]; });
```

I think this is not the right type:

https://github.com/cms-sw/cmssw/blob/b4fcf8686c7705e1123b299b5efaf11ad904ff69/DataFormats/SiStripCluster/interface/SiStripCluster.h#L95